app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/AllSessionsFragment.kt

```diff
@@ -81,6 +84,12 @@ class AllSessionsFragment : Fragment(), Injectable {
         sessionsViewModel.isLoading.observe(this, { isLoading ->
             progressTimeLatch.loading = isLoading ?: false
         })
+        sessionsViewModel.refreshFocusCurrentSession.observe(this, {
+            if (it != true) return@observe
+            val now = Date()
```

Comment:
I think it's better to consider timezone.
The timezone of our session date is JST, so this current date should be converted to JST.
This is example but I haven't checked yet. Can you check it works? 🙇 

```kotlin
val currentJSTDate = Date(ZonedDateTime.now(ZoneId.of("JST")).toInstant().toEpochMilli())
```

Reply:
👀

Reply:
@sho5nn If it doesn't work well, it's okay to use `Calendar`! 🙇 

```kotlin
Calendar cal = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
cal.setTimeInMillis(currentTImeMills);
Date now = cal.getTime();
```

Reply:
It worked!
```diff
-Date(ZonedDateTime.now(ZoneId.of("JST")).toInstant().toEpochMilli())
+Date(ZonedDateTime.now(ZoneId.of(ZoneId.SHORT_IDS["JST"])).toInstant().toEpochMilli())
```